doc/cookbook/source/examples/classifier/linear_svm.rst

```diff
@@ -0,0 +1,60 @@
+=============================
+Linear Support Vector Machine
+=============================
+
+Linear Support Vector Machine is a binary classifier which finds a hyper-plane such that the margins between the two classes are maximized. These margins can be described by the following equations:
+
+.. math::
+    {\bf w}^\top{\bf x} + b = 1
```

Comment:
You could also write the minimization problem. But we only want one major equation